src/lib/hooks/useUpgradeDialog.js

```diff
@@ -45,7 +45,9 @@ const styles = StyleSheet.create({
 const WhatsNewButtonComponent = () => {
   const handlePress = useOnPress(() => window.open(Config.newVersionUrl, '_blank'))
 
-  return (
+  //dont show whats new if just a patch
+  const isPatch = (Config.version.match(/[0-9]+\.[0-9]+\.([0-9]+)/) || [])[0] !== '0'
```

Comment:
1. better to determine isPatch once, in config.js
2. there's shortcut \d for [0-9] match group
3. i don't understand the logic of detection is the update a patch or not. [0] index of regexp match object generally returns the  **whole** substring matched. e.g. 1.5.1. But you comparing it with '0' that means you want to compare a signle digit. How isPatch logic should work ? Pls tell me - i'll help with the regexp checking it